code/modules/mob/living/simple_animal/abnormality/he/happy_teddy_bear.dm

```diff
@@ -66,6 +67,17 @@
 	var/last_pinged = 0
 	var/time_strangled = 0
 	while(user.stat != DEAD)
+		if(time_strangled >= 4 && get_attribute_level(user, FORTITUDE_ATTRIBUTE) >= 60)
+			if(user.stat != UNCONSCIOUS) //Wouldn't make sense if they break free while passed out
+				src.break_check = TRUE
```

Comment:
could ya say why you added `src` to many things here?

Reply:
I copypasted part of it from below so I figured something prevented me from doing it normally I guess?

Reply:
how shitcode spreads 101
please just delete the `src.` part of the vars being changed

Reply:
There, did it throughout that entire file in fact since looking at it, about a good amount of them weren't needed.